src/components/Media.js

```diff
@@ -573,10 +575,60 @@ export default class Video extends Component {
         >
           {this.renderChildren()}
         </video>
+      );
+    }
+    return (
+      <audio
+        className={classNames('video-react-video', this.props.className)}
+        id={videoId}
+        crossOrigin={crossOrigin}
+        ref={c => {
+          this.video = c;
+        }}
+        muted={muted}
+        preload={preload}
+        loop={loop}
+        playsInline={playsInline}
+        autoPlay={autoPlay}
+        poster={poster}
+        src={src}
+        onLoadStart={this.handleLoadStart}
+        onWaiting={this.handleWaiting}
+        onCanPlay={this.handleCanPlay}
+        onCanPlayThrough={this.handleCanPlayThrough}
+        onPlaying={this.handlePlaying}
+        onEnded={this.handleEnded}
+        onSeeking={this.handleSeeking}
+        onSeeked={this.handleSeeked}
+        onPlay={this.handlePlay}
+        onPause={this.handlePause}
+        onProgress={this.handleProgress}
+        onDurationChange={this.handleDurationChange}
+        onError={this.handleError}
+        onSuspend={this.handleSuspend}
+        onAbort={this.handleAbort}
+        onEmptied={this.handleEmptied}
+        onStalled={this.handleStalled}
+        onLoadedMetadata={this.handleLoadedMetaData}
+        onLoadedData={this.handleLoadedData}
+        onTimeUpdate={this.handleTimeUpdate}
+        onRateChange={this.handleRateChange}
+        onVolumeChange={this.handleVolumeChange}
+        tabIndex="-1"
+      >
+        {this.renderChildren()}
+      </audio>
+    );
+  }
+
+  render() {
+    return (
+      <div className={classNames('video-react-video-container')}>
```

Comment:
is classNames doing something tricky here or can we just pass in `video-react-video-container`?

Reply:
That was retained from video-react's original code. I can take a look to see what's going on